source/_data/themes.yml

```diff
@@ -3076,3 +3076,13 @@
     - hexo-theme
     - simple
     - clean
+
+- name: Minimalist
+  description: A minimalist style of pure green Hexo theme
+  link: https://github.com/CN-DXTZ/hexo-theme-Minimalist
+  preview: https://cn-dxtz.github.io
+  tags:
+    - Minimalist/极简
```

Comment:
![image](https://user-images.githubusercontent.com/16578570/75703713-49326a00-5cb8-11ea-8599-2fd523eb30a3.png)

Looks good except for the tags

Reply:
Thanks, I have deleted the error of tags

Reply:
@CN-DXTZ Please avoid using Chinese in tags and descriptions.